api/v1/routers/motorists.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+module.exports = motoristsRouter;
+
+function motoristsRouter(express, validator, expressJoi, motoristsCtrls){
+   var router = express.Router();
+   
+   router.get('/', motoristsCtrls.fetchAll);
+   
+   router.patch('/location/:motorist',expressJoi.joiValidate(validator.userLocation.patch));
+   router.patch('/location/:motorist', motoristsCtrls.setAsDriver);
```

Comment:
We said we are going to chain handlers (separated by comma) instead of registering an entry for each one